rh/management/commands/load_fictive_data.py

```diff
@@ -0,0 +1,550 @@
+  {'name': 'United States', 'code': 'US'}, 
+  {'name': 'United States Minor Outlying Islands', 'code': 'UM'}, 
+  {'name': 'Uruguay', 'code': 'UY'}, 
+  {'name': 'Uzbekistan', 'code': 'UZ'}, 
+  {'name': 'Vanuatu', 'code': 'VU'}, 
+  {'name': 'Venezuela', 'code': 'VE'}, 
+  {'name': 'Viet Nam', 'code': 'VN'}, 
+  {'name': 'Virgin Islands, British', 'code': 'VG'}, 
+  {'name': 'Virgin Islands, U.S.', 'code': 'VI'}, 
+  {'name': 'Wallis and Futuna', 'code': 'WF'}, 
+  {'name': 'Western Sahara', 'code': 'EH'}, 
+  {'name': 'Yemen', 'code': 'YE'}, 
+  {'name': 'Zambia', 'code': 'ZM'}, 
+  {'name': 'Zimbabwe', 'code': 'ZW'} 
+]
+
+CLOCATION = {'parent': None, 'level': 0, 'name': 'Afghanistan', 'code': 'AF', 'original_name': 'افغانستان', 'type': 'Country', 'lat': 0, 'long': 0}
+PLOCATION = [
+    {'level': 1, 'name': 'Kabul', 'code': 'AF01', 'original_name': 'کابل', 'type': 'Province', 'lat': 0, 'long': 0},
+    {'level': 1, 'name': 'Samangan', 'code': 'AF20', 'original_name': 'سمنگان', 'type': 'Province', 'lat': 0, 'long': 0},
+    {'level': 1, 'name': 'Logar', 'code': 'AF05', 'original_name': 'لوگر', 'type': 'Province', 'lat': 0, 'long': 0},
+    {'level': 1, 'name': 'Paktika', 'code': 'AF12', 'original_name': 'پکتیکا', 'type': 'Province', 'lat': 0, 'long': 0},
+    {'level': 1, 'name': 'Sar-e-Pul', 'code': 'AF22', 'original_name': 'سرپل', 'type': 'Province', 'lat': 0, 'long': 0},
+]
+DLOCATION = [
+    {'parent': 'AF01', 'level': 2, 'name': 'Estalef', 'code': 'AF0113', 'original_name': '', 'type': 'District', 'lat': 34.848476, 'long': 69.022911},
+    {'parent': 'AF01', 'level': 2, 'name': 'Paghman', 'code': 'AF0102', 'original_name': '', 'type': 'District', 'lat': 34.541674, 'long': 68.935676},
+    
+    {'parent': 'AF20', 'level': 2, 'name': 'Feroz Nakhchir', 'code': 'AF2004', 'original_name': '', 'type': 'District', 'lat': 36.446451, 'long': 67.610323},
+    {'parent': 'AF20', 'level': 2, 'name': 'Dara-e-Suf-e-Bala', 'code': 'AF2007', 'original_name': '', 'type': 'District', 'lat': 35.640448, 'long': 67/.262889},
+    
+    {'parent': 'AF05', 'level': 2, 'name': 'Khoshi', 'code': 'AF0504', 'original_name': '', 'type': 'District', 'lat': 33.992485, 'long': 69.284904},
+    {'parent': 'AF05', 'level': 2, 'name': 'Mohammad Agha', 'code': 'AF0505', 'original_name': '', 'type': 'District', 'lat': 34.204792, 'long': 69.18032},
+
+    {'parent': 'AF12', 'level': 2, 'name': 'Wormamay', 'code': 'AF1218', 'original_name': '', 'type': 'District', 'lat': 31.932197, 'long': 68.858521},
+    {'parent': 'AF12', 'level': 2, 'name': 'Sharan', 'code': 'AF1201', 'original_name': '', 'type': 'District', 'lat': 33.211648, 'long': 68.74734},
+
+    {'parent': 'AF22', 'level': 2, 'name': 'Balkhab', 'code': 'AF2207', 'original_name': '', 'type': 'District', 'lat': 35.453001, 'long': 35.928227},
+    {'parent': 'AF22', 'level': 2, 'name': 'Gosfandi', 'code': 'AF2206', 'original_name': '', 'type': 'District', 'lat': 33.211648, 'long': 66.54342}
+]
+
+JSON_VALUES = {
```

Comment:
can we have at least twi activities with different json fields ? because right now the form doesn't change when we change the activity and this we cannot test the dynamic fields. 
<img width="499" alt="Screenshot 2022-09-26 at 13 05 30" src="https://user-images.githubusercontent.com/939309/192250037-ae5db118-a525-4d87-8c9c-ab02621a7b69.png">

Reply:
Done